pact/Pact/Core/IR/Eval/CEK.hs

```diff
@@ -1518,7 +1520,8 @@ applyContToValue (ObjC env info currfield fs vs cont) handler v = do
 
 applyContToValue (EnforceErrorC info _) handler v = case v of
   VString err -> returnCEK Mt handler (VError err info)
-  _ -> failInvariant info "enforce function did not return a string"
+  -- TODO: remove
```

Comment:
Leftover TODO?

Reply:
Nope! Good catch. I do plan to cherry pick some changes onto master though